converter/fluka/input.py

```diff
@@ -0,0 +1,76 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Input:
+    """Class mapping of the fluka input file."""
+
+    energy: float = 90.0  # GeV FLUKA specific
```

Comment:
If you decide to keep energy in GeV, then I suggest to encode it in the parameter name, say `energy_GeV`